src/Swarm/TUI/View/CellDisplay.hs

```diff
@@ -58,9 +79,47 @@ displayLoc showRobots g coords =
       then displayRobotCell g coords
       else []
 
-data HideEntity = HideAllEntities | HideNoEntity | HideEntityUnknownTo Robot
+-- | Add random "static" based on the distance to the robot being
+--   @view@ed.
+addStatic :: GameState -> W.Coords -> Display -> Display
+addStatic g coords d = maybe d displayStatic (getStatic g coords)
 
-hidingMode :: GameState -> HideEntity
-hidingMode g
-  | g ^. creativeMode = HideNoEntity
-  | otherwise = maybe HideAllEntities HideEntityUnknownTo $ focusedRobot g
+-- | Draw black or white static.
```

Comment:
I wonder what it would look like with a couple of shades of gray mixed in.  Typically static has a "grayish" appearance, due to the specks being so small and blurred together.  But these specks are large and chunky.  What we lack in spatial resolution in our TUI display, we could make up for in "color depth".

Reply:
For a more granular effect, in the long term we could explore using [half blocks](https://en.wikipedia.org/wiki/Block_Elements) (or even quadrant blocks).

Reply:
Oh, very good idea, and good observation re: gray and spatial vs color resolution.  I remember talking about this with you before, thanks for pointing out the relevant concept again.  Indeed, I thought it needed more gray, so I tried changing  to use light & dark gray instead of black & white, but that looked bad (the gray blended in with any `stone` texture in the map).  But maybe using gray in combination with black and white, and/or using more granular blocks, could help make it look better.

Reply:
#1145